zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowModel.java

```diff
@@ -144,4 +182,20 @@ public static Builder builder() {
     }
   }
 
+  private static MetaGraphDef extractMetaGraphDefinition(final SavedModelBundle bundle) {
+    final MetaGraphDef metaGraphDef;
+    try {
+      metaGraphDef = MetaGraphDef.parseFrom(bundle.metaGraphDef());
+    } catch (InvalidProtocolBufferException e) {
+      throw new RuntimeException("Failed parsing tensorflow metagraph definition", e);
```

Comment:
🎨 small improvement: I think it would be better to throw a custom checked exception and handle it where it's used like in the create method.